spdm.rflx

```diff
@@ -331,11 +383,63 @@ package SPDM is
          --  be derived from the measurement hash algorithm that the ALGORITHM response message
          --  returns.
          --  FIXME: What does "shall be derived" mean - equals the hash length?
-         DMTF_Spec_Measurement_Value_Size : Length;
+         DMTF_Spec_Measurement_Value_Size_1 : Length;
+         DMTF_Spec_Measurement_Value_Size_2 : Length;
          DMTF_Spec_Measurement_Value : Opaque
-            with Size => 8 * DMTF_Spec_Measurement_Value_Size
+            with Size => 8 * (DMTF_Spec_Measurement_Value_Size_2 * 256 + DMTF_Spec_Measurement_Value_Size_1)
             then null
-               if Measurement_Size = 8 * (DMTF_Spec_Measurement_Value'Last - Measurement_Size'Last);
+               if 8 * (Measurement_Size_2 * 256 + Measurement_Size_1)
+                  = DMTF_Spec_Measurement_Value'Last - Measurement_Size_2'Last;
+      end message;
+
+   type Measurement_Record is sequence of Measurement_Block;
+
+   type Measurement_Count is range 0 .. 2 ** 8 - 1 with Size => 8;
+
+   type Block_Count is range 0 .. 2 ** 8 - 1 with Size => 8;
+
+   type Measurements_Response_1_0 is
+      message
+         Number_Of_Indices : Measurement_Count;
+         Reserved_1 : Reserved_8;
+         Number_Of_Blocks : Block_Count;
+         Measurement_Record_Length_1 : Length;
+         Measurement_Record_Length_2 : Length;
+         Measurement_Record_Length_3 : Length;
+         Measurement_Record : Measurement_Record
+            with Size => 8 * (Measurement_Record_Length_3 * 65536
+                              + Measurement_Record_Length_2 * 256
+                              + Measurement_Record_Length_1);
+         Nonce : Opaque
+            with Size => 8 * 32;
+         Opaque_Length_1 : Length;
+         Opaque_Length_2 : Length;
+         Opaque_Data : Opaque
+            with Size => 8 * (Opaque_Length_2 * 256 + Opaque_Length_1);
+         -- Signature : Signature;  --  Conditional
+      end message;
+
+   type Measurements_Response_1_1 is
+      message
+         Number_Of_Indices : Measurement_Count;
+         Reserved_1 : Reserved_4;
+         Slot_ID : Narrow_Slot;
+         Number_Of_Blocks : Block_Count;
+         Measurement_Record_Length_1 : Length;
+         Measurement_Record_Length_2 : Length;
+         Measurement_Record_Length_3 : Length;
+         Measurement_Record : Measurement_Record
+            with Size => 8 * (Measurement_Record_Length_3 * 65536
+                              + Measurement_Record_Length_2 * 256
+                              + Measurement_Record_Length_1);
+         Nonce : Opaque
+            with Size => 8 * 32;
+         Opaque_Length_1 : Length;
+         Opaque_Length_2 : Length;
+         Opaque_Data : Opaque
+            with Size => 8 * (Opaque_Length_2 * 256 + Opaque_Length_1);
+         Signature : Opaque
```

Comment:
This should be marked with `ISSUE` and an explanation that the presence depends on the previous measurement request.

Reply:
Fixed by https://github.com/Componolit/RecordFlux-specifications/pull/82/commits/99eb827f3ff9fa2bc589dde2b14be99cf2ca341d.